### PR TITLE
V3 feedback

### PR DESCRIPTION
**Pull request by cpvalente**
small tweaks from user feedback on alpha.3

- [x] rename `general` -> `app settings`
- [x] close `param editor` on submit
- [x] rename buttons in `param editor`
- [x] fix blackout
- [x] rename `message blink` > `blink`
- [x] rename `readonly` -> `read only`
- [x] always show all links in navigation menu
- [x] issue with `offset` showing as positive in `time to end` timers
- [x] show pincode for operator

### CODE REVIEW

File: apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx [hunk 1]
```diff
@@ -96,10 +99,7 @@ export default function ViewParamsEditor({ paramFields }: EditFormDrawerProps) {
...
         <DrawerFooter className={style.drawerFooter}>
           <Button variant='ontime-ghosted' onClick={resetParams} type='reset'>
-            Reset
-          </Button>
-          <Button variant='ontime-subtle' onClick={onCloseWithoutSaving}>
-            Cancel
+            Reset to default
           </Button>
           <Button variant='ontime-filled' form='edit-params-form' type='submit'>
             Save
```
Comment (alex-Arc) [inline apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx:105]: did you want to call this submit?

**Review by alex-Arc — APPROVED** [review]

File: apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx (hunk 1 above)
Comment (cpvalente) [reply apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx:105]: I didn't understand the question.. is there something with the naming?

Comment (cpvalente) [reply apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx:105]: Ah yes, sorry.

I tried it and it felt a bit weird. 
The idea of submit was to clarify the expectation of the window disappearing on click, but I think it made the language more complicated.
What do you think?

Comment (alex-Arc) [reply apps/client/src/common/components/view-params-editor/ViewParamsEditor.tsx:105]: both are OK with me
if you think its a bit wired let's not do it

**Pull request merged**